xml/System.Runtime.InteropServices/CollectionsMarshal.xml

```diff
@@ -281,14 +281,24 @@ The ref `null` can be detected by calling <xref:System.Runtime.CompilerServices.
         <Parameter Name="key" Type="TAlternateKey" Index="1" FrameworkAlternate="net-9.0" />
       </Parameters>
       <Docs>
-        <typeparam name="TKey">To be added.</typeparam>
-        <typeparam name="TValue">To be added.</typeparam>
-        <typeparam name="TAlternateKey">To be added.</typeparam>
-        <param name="dictionary">To be added.</param>
-        <param name="key">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <typeparam name="TKey">The type of the keys in the dictionary.</typeparam>
+        <typeparam name="TValue">The type of the values in the dictionary.</typeparam>
+        <typeparam name="TAlternateKey">The type of an alternate key for lookups in the dictionary.</typeparam>
+        <param name="dictionary">The dictionary to get the ref to <typeparamref name="TValue"/> from.</param>
+        <param name="key">The key used for lookup.</param>
+        <summary>Gets either a ref to a <typeparamref name="TValue"/> in the <see cref="T:System.Collections.Generic.Dictionary`2"/> or a ref null if it does not exist in the <paramref name="dictionary"/>.</summary>
+        <returns>A reference to a <typeparamref name="TValue" /> in the <see cref="T:System.Collections.Generic.Dictionary`2" /> or a reference <see langword="null" /> if it does not exist in the <paramref name="dictionary" />..</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Items should not be added or removed from the <xref:System.Collections.Generic.Dictionary`2> while the ref `TValue` is in use.
+
+The ref `null` can be detected by calling <xref:System.Runtime.CompilerServices.Unsafe.IsNullRef%60%601(%60%600@)?displayProperty=fullName>.
+
+          ]]></format>
+        </remarks>
```

Comment:
Did the porting tool do this? It's supposed to port remarks as plain xml, not as markdown anymore (for simplicity).

```suggestion
        <remarks>
          <para>Items should not be added or removed from the <see cref="T:System.Collections.Generic.Dictionary`2" /> while the ref <paramref name="TValue" /> is in use.</para>
          <para>The ref <see langword="null" /> can be detected by calling <see cref="M:System.Runtime.CompilerServices.Unsafe.IsNullRef``1(``0@)" />.</para>
        </remarks>
```

Reply:
This was copied manually from the above method's docs as I couldn't get the porting tool to work right for CoreLib

Reply:
> as I couldn't get the porting tool to work right for CoreLib

Oh, what command did you use? The xml docs for S.P.CoreLib are found under `artifacts\bin\coreclr\<os>.<arch>.<config>\IL\`, that would be the argument for `-IntelliSense`.

Reply:
I did that, but the updated triple slash comments weren't being output into that folder for some reason. I didn't feel like fighting it.

Reply:
Did you also run `build.cmd clr.corelib` before porting?

Reply:
Yep. Even deleted my bin artifacts. Probably should have deleted my obj ones as well.

Reply:
I had the same problem with System.Runtime.InteropServices initially and solved it by using the obj folder instead. Couldn't get that to work with CoreLib though.